<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_dialog</class>
 <widget class="QDialog" name="edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>511</width>
    <height>335</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Dataset</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/icons.qrc">
    <normaloff>:/images/meta.png</normaloff>:/images/meta.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>outcomes</string>
   </property>
   <widget class="QListView" name="outcome_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>209</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_outcome_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/add_48.png</normaloff>:/images/function_icon_set/add_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_outcome_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/cancel_48.png</normaloff>:/images/function_icon_set/cancel_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>151</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>tx groups</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListView" name="group_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_group_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/add_48.png</normaloff>:/images/function_icon_set/add_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_group_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/cancel_48.png</normaloff>:/images/function_icon_set/cancel_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpbox_follow_ups">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>151</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>follow ups</string>
   </property>
   <widget class="QListView" name="follow_up_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>209</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_follow_up_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/cancel_48.png</normaloff>:/images/function_icon_set/cancel_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="add_follow_up_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/icons.qrc">
      <normaloff>:/images/function_icon_set/add_48.png</normaloff>:/images/function_icon_set/add_48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>edit_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>edit_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
